src/components/CivicSubscriptionView/CivicSubscriptionView.vue

```diff
@@ -325,6 +327,10 @@ export default {
       type: Boolean,
       default: false,
     },
+    initialState: {
+      type: String,
+      default: undefined,
```

Comment:
Strange decision to not set a default value for state, setting one should always be desirable?
Or maybe we need a state representing "undefined"